polaris/run/serial.py

```diff
@@ -342,23 +344,30 @@ def _run_task(task, available_resources):
     cwd = os.getcwd()
     for step_name in task.steps_to_run:
         step = task.steps[step_name]
-        step_start = time.time()
+        complete_filename = os.path.join(step.work_dir,
+                                         'polaris_step_complete.log')
+
+        _print_to_stdout(task, f'  * step: {step_name}')
 
+        if os.path.exists(complete_filename):
+            _print_to_stdout(task, '          already completed')
+            continue
```

Comment:
I noticed that the user doesn't get a message like this to stdout if they use `polaris serial` in the step directory. Should we add something similar to `step.run`?

Reply:
Yes, great point.  I noticed that, too.

Reply:
I have added more output when steps are run on their own:
```
$ polaris serial
polaris calling: polaris.run.serial._run_task()
  in /gpfs/fs1/home/ac.xylar/e3sm_work/polaris/reorg-for-shared-steps/polaris/run/serial.py

Running step: init
  * step: init

....

          execution:        SUCCESS
          runtime:          0:00:04
```
Running again produces:
```
$ polaris serial
polaris calling: polaris.run.serial._run_task()
  in /gpfs/fs1/home/ac.xylar/e3sm_work/polaris/reorg-for-shared-steps/polaris/run/serial.py

Running step: init
  * step: init
          already completed
```

Reply:
Awesome. Thanks!